modules/simple_forms_api/app/services/simple_forms_api/notification_email.rb

```diff
@@ -68,13 +69,14 @@ def initialize(config, notification_type: :confirmation, user: nil)
       @lighthouse_updated_at = config[:lighthouse_updated_at]
       @notification_type = notification_type
       @user = user
+      @user_account = user_account
     end
 
     def send(at: nil)
       return unless SUPPORTED_FORMS.include?(form_number)
 
       data = form_specific_data || empty_form_specific_data
-      return if data[:email].blank? || data[:personalization]['first_name'].blank?
+      return if data[:personalization]['first_name'].blank?
```

Comment:
Does it not matter if email is blank any longer?

Reply:
Thanks for taking a look so quickly! In cases where we have a `User` or `UserAccount` then yeah it doesn't matter because we'll grab email off those records. And when we don't I've moved the check for `data[:email].blank?` to the appropriate spot (lines 112 and 133) just before sending an email.